tensorflow/core/util/mkl_util.h

```diff
@@ -1886,9 +1972,18 @@ class MklPrimitiveFactory {
  private:
   static inline LRUCache<MklPrimitive>& GetLRUCache() {
     static const int kCapacity = 1024;  // cache capacity
+#ifndef DNNL_AARCH64_USE_ACL
     static thread_local LRUCache<MklPrimitive> lru_cache_(kCapacity);
+#else
+    static LRUCache<MklPrimitive> lru_cache_(kCapacity);
```

Comment:
Please add thread safety annotations. Same for `in_flight_`.
```suggestion
    static LRUCache<MklPrimitive> lru_cache_(kCapacity); TF_GUARDED_BY(lru_mu_)
```